owl/EASE.owl

```diff
@@ -1144,6 +1189,42 @@ The second issue relates to what is &quot;outside&quot;. The state is a configur
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE.owl#3DPosition -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE.owl#3DPosition">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE.owl#EuclideanPoint"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ease-crc.org/ont/EASE.owl#hasTranslation"/>
+                <owl:someValuesFrom rdf:resource="http://www.w3.org/2001/XMLSchema#string"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:comment>A point in three dimensional space, given as translation.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE.owl#6DPose -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE.owl#6DPose">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE.owl#EuclideanPoint"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ease-crc.org/ont/EASE.owl#hasRotation"/>
+                <owl:someValuesFrom rdf:resource="http://www.w3.org/2001/XMLSchema#string"/>
```

Comment:
As before, we should consider whether we need more rotations/translations here (and a "some" restriction) or whether we have an "exactly 1" restriction.

Reply:
Yes, the idea was that you could have assertions with different units of measurement. I will change it to exactly 1 for